bspump/mqtt_metrics/__init__.py

```diff
@@ -136,11 +145,19 @@ def on_message(self, client, userdata, message):
                 L.warning(f"Payload sent to {topic} is not a valid JSON. Payload: {payload}")
                 return
             
-            method = payload.get("method")
-            if method is not None and method == "get":
+            topology_count = payload.get("topology_count")
+            if topology_count > 2000:
```

Comment:
If you use `topology_count = min(topology_count, 2000)` then you don't end up repeating the 2000 which makes it easier to change that number in the future.

Reply:
Okay cool, thanks. That will look prettier